requests-for-comments/2022-11-09-nail-distribution.md

```diff
@@ -0,0 +1,71 @@
+# Request for comments: Nail Content Distribution
+
+## Problem statement
+
+We don't have a clear view on how to maximize the impact of our content through good distribution. We have tried (and are trying) various initiatives, but it's not particularly coordinated or structured, so it's hard to say what's definitely working well and what's not for the most part. 
+
+It's rational to go deep on a smaller selection of distribution channels and trying to do them really well, rather than blasting out content onto every platform imaginable in a shallow way. We've found this quality > quantity approach works well for us when it comes to writing the content itself. 
+
+Another problem is that it's not clear whose responsibility on the team content distribution is, so it ends up being a bit ad hoc. This is easy to solve though. 
+
+## Context
+
+These are the places where we distribute content right now, and who does it:
+
+- PostHog LinkedIn - _Andy mainly, sometimes Joe or James H_
+- PostHog Twitter - _Andy mainly, not sure if anyone else does?_
+- James Hawkins' personal Twitter - _James H_
+- HogMail - _Andy_
+- User Slack - _random, usually the content author_
+- Hacker News - _random, usually the content author_
+- SEO (not really a 'distribution' channel per se, but included for completeness) - _Andy_
+
+- LinkedIn paid ads - _Charles, via Hey_
+- Twitter paid ads - _Andy, via Hey_
+- Various newsletter sponsorships - _Charles_
+
+These are the places we want to try distributing content in Q4:
+
+- TikTok - _James H, via Flowlync_
+- YouTube - _Ian_
+
+These are the places we have also considered and decided not to try for now:
+
+- Reddit
+- Other Slacks
+- Other Discords
+- Hacker Noon, Dev To, IndieHackers
+
+It's _a lot._ And it's not obvious which ones we really care about doing well at. 
+
+## Proposal
+
+I'm going to split this into 'let's get good at this channel' vs. 'we should post here out of habit, but we're not trying to win here'. I think we should try a different approach to ownership of these channels though (oo, controversial):
+
+### Let's get (or stay) good at this in Q4
+
+- SEO - _Andy_
+- HogMail - _Andy_
+- James' personal Twitter - _James H_
+- TikTok - _James H_
+- YouTube - _Ian_
+- Charles' personal LinkedIn (maybe) - _Charles_
+
+### Let's keep posting here out of habit/hygiene factor
+
+For _these_ channels, I think it should just be the responsibility of the person who created the content itself to post it in the above channels (or not - it's your call). That way, we're staying focused on the stuff we want to get good at.
+
+- PostHog LinkedIn
+- PostHog Twitter
+- User Slack
+- Hacker News
+- LinkedIn paid ads
+- Twitter paid ads
+
+## How will we measure success?
+
+I didn't get this far yet as I'd like feedback on the above first! But instinctively, it makes sense for each channel owner to just decide this themselves, rather than me being prescriptive about it. 
```

Comment:
Again, I think this is a reason distro should sit with the editor. The editor can then be responsible for having a cohesive view of distro channels, setting benchmarks and keeping context. 

Without this, I think we'd struggle to get repeatable results because there are so many hands at the wheel.